org-noter/org-noter
# Feature/sync noter document

## Problem

If you change the filename of a document, then the notes and document become unlinked.


## Solution

This PR implements issue #40 in the case where the document filename change is performed in `dired-mode` or `wdired-mode`.  This handles any name change, but if the user moves the document to a path location above the notes file, a warning is issued.  `org-noter` will not be able to find the notes file from the document file, so future doc name changes won't be tracked in the notes file, nor can a session be initiated from the doc file.

## Checklist


## Steps to Test

1. With `dired`, open a directory containing a noter-annotated document.
2. Change the name of the file (with `R` or `C-x C-q`).
3. Check the `NOTER_DOCUMENT` property for the document's entry in the notes file.
4. If the notes file has the same basename as the doc file, the notes file's name will also change.

## potential pitfall

If one swaps filenames in `wdired` (ie, `fileA` -> `fileB`; `fileB` -> `fileA`) or does any other sort of cyclic renaming, this feature might not work (I haven't tested it yet).
